app/models/retirement_manager.rb

```diff
@@ -0,0 +1,12 @@
+class RetirementManager
+  def self.check
+    ems_ids = MiqServer.my_server.zone.ext_management_system_ids
+    [LoadBalancer, OrchestrationStack, Vm, Service].each do |model|
```

Comment:
would be nice if we had a dynamic way of asking for `classes_that_support_retirement`